src/types/function.rs

```diff
@@ -3,15 +3,52 @@ use crate::exceptions::PyValueError;
 use crate::prelude::*;
 use crate::{
     class::methods::{self, PyMethodDef},
-    ffi, AsPyPointer,
+    ffi, types, AsPyPointer,
 };
+use std::os::raw::c_void;
 
 /// Represents a builtin Python function object.
 #[repr(transparent)]
 pub struct PyCFunction(PyAny);
 
 pyobject_native_type_core!(PyCFunction, ffi::PyCFunction_Type, #checkfunction=ffi::PyCFunction_Check);
 
+const CLOSURE_CAPSULE_NAME: &[u8] = b"pyo3-closure\0";
+
+fn closure_capsule_name() -> *const i8 {
```

Comment:
Why is this a `fn`? Also, `char` is either `i8` or `u8` depending on platform, so it's easiest to use

`CLOSURE_CAPSULE_NAME.as_ptr() as *const _`

inline, see e.g. https://github.com/PyO3/pyo3/blob/main/src/derive_utils.rs#L311

Reply:
Thanks, I've moved to your suggestion, the goal of the function was to avoid repeating the 'as_ptr()' bit but it certainly not a big gain (and might not be very idiomatic).